modules/ignition/variables.tf

```diff
@@ -196,3 +196,8 @@ variable "nfs_config_file" {
 variable "proxy_exclusive_units" {
   type = "string"
 }
+
+variable "datadog_api_key" {
```

Comment:
I think we will also need another variable definition in the platform folder on the module ignition (or however it plugs together)